force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +275,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
+        Integer sumOfPostiveNums = 0;
 
         // Loop through the list of integers
             // if the number is negative skip this iteration
+        for (Integer num : numbers) {
+            if (num > 0) {
+                sumOfPostiveNums += num;
+            } else {
+                continue;
+            }
+        }
```

Comment:
Fun fact: Zero is usually considered even and positive in mathematics, but probably depends on who you ask.
```suggestion
        for (Integer num : numbers) {
            if (num > 0) {
                sumOfPostiveNums += num;
            }
        }
```